packages/node-http-handler/src/set-socket-keep-alive.ts

```diff
@@ -0,0 +1,22 @@
+import { ClientRequest } from "http";
+
+export interface SocketKeepAliveOptions {
+    keepAlive: boolean;
+    keepAliveMsecs?: number;
+}
+
+export const setSocketKeepAlive = (
+    request: ClientRequest,
+    socketKeepAlive: SocketKeepAliveOptions
+) => {
+    if (!(socketKeepAlive?.keepAlive === true)) {
```

Comment:
Simplify to `!==` check

```suggestion
    if (socketKeepAlive?.keepAlive !== true) {
```